monitor/flowgen/bridging/bridging.go

```diff
@@ -0,0 +1,85 @@
+package bridging
+
+import (
+	"math/big"
+	"time"
+
+	"github.com/omni-network/omni/contracts/bindings"
+	"github.com/omni-network/omni/e2e/app/eoa"
+	"github.com/omni-network/omni/lib/contracts/solvernet"
+	"github.com/omni-network/omni/lib/errors"
+	"github.com/omni-network/omni/lib/netconf"
+	"github.com/omni-network/omni/monitor/flowgen/types"
+	"github.com/omni-network/omni/solver/app"
+
+	"github.com/ethereum/go-ethereum/common"
+)
+
+// NewJob instantiates the flow-specific job configuration.
```

Comment:
not sure what `flow-specific` mean? Maybe something like: `NewJob returns a job that bridges blah blah blah`